CaPPMS/Pages/Faq.razor

```diff
@@ -1,7 +1,203 @@
-﻿@page "/faq"
+﻿@using System.Collections.Concurrent;
+@using CaPPMS.Data
+@using CaPPMS.Model
+
+@using System.Collections.Generic;
+@using System.IO;
+@using System.Security.Principal;
+@using System.Threading.Tasks;
+
+@using Newtonsoft.Json;
+@using Octokit;
+@using iTextSharp.text;
+@using iTextSharp.text.pdf;
+@using System;
+
+@inject FaqManagerService faqManager
+
+
+@page "/faq"
+<head>
+    <style>
+        table, th, td {
+            border: 1px solid black;
+        }
+        .intro {
+            color:black;
+            font-weight:bold;
+        }
+    </style>
+
+   
+</head>
+<h3>FAQs</h3>
+
+
+ <div class="intro">
+        The purpose of this page is for users to ask questions regarding the application as well as review previously asked questions.
+
+        Currently the form only allows users to input a question and not a reply to other users questions because we do not want just anyone to be able to respond to the questions.
+
+        The questions need to be reviewed by the administrator and then answered in order to maintain accuracy and professionalism. 
+ </div>
+
+<div class="m-2 p-2 rounded">
+    @if (NewFaq != null)
```

Comment:
This is probably not needed

Reply:
remove everything that is in this block?

Reply:
No, no... just the if statement right? NewFaq I believe gets initialized on load so it will never be null and thus we don't have to waste the cpu cycle to check.